src/Phinx/Migration/CreationInterface.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Phinx
+ *
+ * (The MIT license)
+ * Copyright (c) 2014 Rob Morgan
```

Comment:
New files get new date, other files get date updated as they are touched. No need to global search and replace. :)